source/common/http/access_log/access_log_formatter.cc

```diff
@@ -17,39 +17,73 @@ const std::string FilterReasonUtils::UPSTREAM_CONNECTION_TERMINATION = "UC";
 const std::string FilterReasonUtils::UPSTREAM_OVERFLOW = "UO";
 const std::string FilterReasonUtils::NO_ROUTE_FOUND = "NR";
 const std::string FilterReasonUtils::FAULT_INJECTED = "FI";
+const std::string FilterReasonUtils::DELAY_INJECTED = "DI";
```

Comment:
We could either have all of the changes in one PR (that's harder to review, too many changes)
or have one change for refactoring (this change, and remove DI) and one change with fully DI support.

I'd like to have two clean changes. Let me know if any concerns.